blockchains/algorand/assets/31566704/info.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "USD Coin",
+    "website": "https://www.centre.io",
+    "description": "USDC provides a fully collateralized US dollar stablecoin, and is based on the open source asset-backed stablecoin framework developed by Centre.",
+    "explorer": "https://algoexplorer.io/asset/31566704",
+    "research": "https://research.binance.com/en/projects/usd-coin",
+    "type": "ALGORAND",
```

Comment:
ASA

Reply:
@vcoolish @George2181 @cryptomanz1 should it be `ASA` [there also? (link)](https://github.com/trustwallet/assets/blob/master/blockchains/algorand/assets/137594422/info.json#L8)

Reply:
așa is only for algorand.
stands for algorand standard asset